pkg/stack/stack_processor.go

```diff
@@ -275,7 +280,11 @@ func ProcessConfig(
 
 	// Terraform section
 	if i, ok := globalTerraformSection["vars"]; ok {
-		terraformVars = i.(map[interface{}]interface{})
+		terraformVars, ok = i.(map[interface{}]interface{})
+
+		if !ok {
+			return nil, errors.New(fmt.Sprintf("Invalid 'terraform.vars` section in the stack '%s'", stack))
```

Comment:
```suggestion
			return nil, errors.New(fmt.Sprintf("Invalid `terraform.vars` section in the stack '%s'", stack))
```

It would be a big help if we could specify which component this is for.